kotlin-mui/src/main/generated/mui/material/TableCell.ext.kt

```diff
@@ -2,6 +2,17 @@
 
 package mui.material
 
+@Suppress("NAME_CONTAINS_ILLEGAL_CHARS")
+// language=JavaScript
+@JsName("""(/*union*/{asc: 'asc', desc: 'desc', false: 'false'}/*union*/)""")
```

Comment:
It is `false` (`bool`, not `string`) in original.
https://mui.com/api/table-cell/

Will it be changed in future?

Reply:
Reproduced with ``` SortDirection.`false` ```

![image](https://user-images.githubusercontent.com/32072831/148366981-d355f237-5e18-4c00-9597-1d36a1e8ac6c.png)